app/models/vendor.rb

```diff
@@ -0,0 +1,17 @@
+class Vendor < ActiveRecord::Base
+  belongs_to :market
+  has_many :products
+  has_many :sales
+
+  def self.current_month
+    Time.now.month
+  end
```

Comment:
I think we had issues with accessing the helper methods from inside the console rather than in the erb templates. In restorspect, though, I'm guessing this is an issue with how we were calling the method / we needed to `include` the module in the controller?